kansha/card_addons/checklist/comp.py

```diff
@@ -101,8 +101,16 @@ def set_done(self):
             data)
 
 
-class Checklist(object):
+class ChecklistItemTask(component.Task):
+    def __init__(self):
+        self.comp = NewChecklistItem()
+
+    def go(self, comp):
+        comp.call(self.comp, 'button')
+        comp.answer(comp.call(self.comp, 'edit'))
+
```

Comment:
This changes the behavior as defined in #186: once opened, the input should remain so, for easier typing of many items.